packages/devtools_app/test/memory/shared/heap/heap_analyzer_golden_test.dart

```diff
@@ -2,13 +2,38 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:io';
```

Comment:
if we are adding a `dart:io` import, we should also add `@TestOn('vm')` since this test would no longer be able to be run on web, (as long as we are okay with this test no longer running on web)

Reply:
on second thought, maybe we should put this memory foot print test in another test library? having designated memory tests like this for DevTools is something we will likely want to do for other places where we suspect memory bloat.

Wdyt about creating
`test/memory/memory_footprint_test.dart` where we can put all tests for the memory page that are related to monitoring memory consumption of Devtools itself?

Reply:
good ideas!